package.json

```diff
@@ -33,6 +33,8 @@
     "sails-disk": "0.10.x"
   },
   "devDependencies": {
-    "lab": "5.x.x"
+      "eslint-config-hapi": "^9.1.0",
```

Comment:
It's small shit, but the spacing is now inconsistent in here.  Can we make these lines have 2spc rather than 4spc?